ucs/adapters/inbound/fastapi_/models.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 - 2022 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""REST API-specific data models (not used by core package)"""
+
+from datetime import datetime
+from typing import Literal, Optional
+
+from pydantic import BaseModel, Field
+
+
+class FileMetadata(BaseModel):
+    """
+    A model containing all the metadata needed to pass it on to other microservices
+    """
+
+    file_id: str
+    file_name: str
+    md5_checksum: str
+    size: int
+    grouping_label: str
+    creation_date: datetime
+    update_date: datetime
+    format: str
+    current_upload_id: Optional[str] = Field(
+        None,
+        description="ID of the current upload. `Null` if no update has been initiated, yet.",
+    )
+
+
+class UploadCreation(BaseModel):
+    """Properties required to create a new upload."""
+
+    file_id: int = Field(
```

Comment:
This is defined as string in the epic docs: https://github.com/ghga-de/epic-docs/blob/6281664e748339febdf53221e9c3ff1c5c5da733/4-star-nosed_mole/api_definitions/rest/ucs.yaml#L170-L172

Reply:
Thanks of course that is right.